### PR TITLE
Better Seed Extractor UI

### PR DESCRIPTION
**Pull request by MTandi**
## About The Pull Request

Added icons to the seed extractor, indicators that show the stat value relative to maximum possible value, tooltips to the stats, list of traits and reagents.

image

There is also an ability to sort and filter the table now:

https://user-images.githubusercontent.com/3625094/201498368-9d620f99-28d3-4197-b73e-cd65d8285334.mp4

And an ability to scrap unneeded seeds:

https://user-images.githubusercontent.com/3625094/202563846-420f395b-d4a0-4ebc-88a9-df74c9c8ca1c.mp4

## Why It's Good For The Game

It wasn't very informative with just numbers, especially if you don't know is it 5 out of 10 or 5 out of 100.

## Changelog

:cl:
qol: new seed extractor UI 
/:cl:

### CODE REVIEW

**Review by MrMelbert — COMMENTED** [review]
Sad not typescript - but I've been looking for a nicer TGUI seed extractor for a while

**Review by MrMelbert — COMMENTED** [review]
Also if you wanna add a tiny bit of style, throw an icon onto the buttons
image

**Comment by DAKKA-WAAAGH** [comment]
This is beautiful.

**Comment by Boopideedoo** [comment]
Could you do the same for the portable version aswell?

**Comment by MTandi** [comment]
> Could you do the same for the portable version aswell?

What? There is a portable version?

**Comment by Boopideedoo** [comment]
> > Could you do the same for the portable version aswell?
> 
> What? There is a portable version?

Yea its used for ghost roles and is craftable I belive its crafted at the autolathe could be wrong about the autolathe being used but its something thats been a big hit with ghost roles

**Comment by MTandi** [comment]
> Yea its used for ghost roles and is craftable I belive its crafted at the autolathe could be wrong about the autolathe being used but its something thats been a big hit with ghost roles

Please tell me the object path. I only know that there is a bag that can extract seeds, but it doesn't have an interface.

**Comment by Boopideedoo** [comment]
> > Yea its used for ghost roles and is craftable I belive its crafted at the autolathe could be wrong about the autolathe being used but its something thats been a big hit with ghost roles
> 
> Please tell me the object path. I only know that there is a bag that can extract seeds, but it doesn't have an interface.

Thats what I was talking about was the bag

**Comment by MTandi** [comment]
> Thats what I was talking about was the bag

It doesn't have a UI, so no. Maybe you need to add a seed extractor to the location of the said ghostrole. It's essentially a stationary back, so there's no balance concern.

File: code/modules/hydroponics/plant_genes.dm [hunk 1]
```diff
@@ -175,21 +177,21 @@
...
 		RegisterSignal(our_plant, COMSIG_PARENT_EXAMINE, PROC_REF(examine))
-
 	return TRUE
 
 /// Add on any unique examine text to the plant's examine text.
 /datum/plant_gene/trait/proc/examine(obj/item/our_plant, mob/examiner, list/examine_list)
 	SIGNAL_HANDLER
 
-	examine_list += examine_line
+	examine_list += "<span class='info'>[description]</span>"
```
Comment (dragomagol) [inline code/modules/hydroponics/plant_genes.dm:188]:
```suggestion
	examine_list += span_info("[description]")
```

File: code/modules/hydroponics/unique_plant_genes.dm [hunk 2]
```diff
@@ -456,6 +473,8 @@
 /// Traits for plants with a different base max_volume.
 /datum/plant_gene/trait/modified_volume
 	name = "Deep Vesicles"
+	description = "It has more reagents than usually."
```
Comment (dragomagol) [inline code/modules/hydroponics/unique_plant_genes.dm:476]:
```suggestion
	description = "It has more reagents than usual."
```

**Pull request merged**